cert.go

```diff
@@ -95,7 +95,7 @@ func (m *mkcert) makeCert(hosts []string) {
 		fatalIfErr(err, "failed to save certificate key")
 	} else {
 		domainCert, _ := x509.ParseCertificate(cert)
-		pfxData, err := pkcs12.Encode(rand.Reader, priv, domainCert, []*x509.Certificate{m.caCert}, "changeit")
+		pfxData, err := pkcs12.Encode(rand.Reader, priv, domainCert, []*x509.Certificate{m.caCert}, "123456")
```

Comment:
I don't think we can change the password as there are already lots of installs using `changeit`.